### PR TITLE
ci: bump maven wrapper to 3.8.7

### PR DESCRIPTION
**Pull request by yeikel**
This pull request was generated using `mvn wrapper:wrapper -Dmaven=3.8.7`

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -1,117 +0,0 @@
-/*
- * Copyright 2007-present the original author or authors.
```
Comment (yeikel) [inline .mvn/wrapper/MavenWrapperDownloader.java:2]: This file is no longer needed

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -5,14 +5,14 @@
 # to you under the Apache License, Version 2.0 (the
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
-# 
-#   http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
```
Comment (pjfanning) [inline .mvn/wrapper/maven-wrapper.properties:9]: this URL should not be changed - some code scanners expect the http form of the URL

this URL redirects to https anyway

Comment (yeikel) [reply .mvn/wrapper/maven-wrapper.properties:9]: Such code scanners aren't doing anyone a favor. How's http better than https? 

Also, I didn't change this myself. This pull request was automatically generated

**Comment by cowtowncoder** [comment]
@pjfanning You are ok with this now?

**Comment by pjfanning** [comment]
@cowtowncoder It's fine as is - as @yeikel points out, this file is generated by mvnw

I volunteer with the ASF and generally, it's discouraged to modify the URL because the official license text has it in http form - but if there is an issue here, it is in mvnw - so let's just ignore the issue, for now.

**Review by cowtowncoder — APPROVED** [review]

**Pull request merged**